tools/serverpod_cli/lib/src/serverpod_packages_version_check/serverpod_packages_version_check.dart

```diff
@@ -0,0 +1,123 @@
+import 'dart:io';
+
+import 'package:pubspec_parse/pubspec_parse.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/util/pubspec_helpers.dart';
+import 'package:source_span/source_span.dart';
+import 'package:yaml/yaml.dart';
+
+class ServerpodPackagesVersionCheckWarnings {
+  static const incompatibleVersion =
+      'Version not compatible with the running serverpod cli version. This '
```

Comment:
`The package version does not match the version of the Serverpod 's CLI. This may cause errors or unexpected behavior.`